src/ContentValidationListenerFactory.php

```diff
@@ -10,12 +10,14 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ContentValidationListenerFactory
+class ContentValidationListenerFactory implements FactoryInterface
```

Comment:
The other option would have been to remove the second and third arguments. However, I'm fine with the fix; I forgot I'd kept `createService()` for BC purposes.